addons/binding/org.openhab.binding.knx/OSGI-INF/SerialBridgeThingHandler.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
How else should this be done with Declarative Services? in KNXHandlerFactory? and what if then new KNXTypeMappers would be registered....?